csharp/ParquetSharp.csproj

```diff
@@ -19,16 +19,13 @@
     <Description>ParquetSharp is a .NET library for reading and writing Parquet files. It's implemented in C# as a PInvoke wrapper around apache-parquet-cpp to provide high performance and compatibility.</Description>
     <Copyright>Copyright G-Research 2021</Copyright>
     <PackageProjectUrl>https://github.com/G-Research/ParquetSharp</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/G-Research/ParquetSharp.git</RepositoryUrl>
     <PackageTags>apache parquet gresearch g-research .net c#</PackageTags>
-    <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
+    <PackageLicenseExpression>apache-2.0</PackageLicenseExpression>
     <PackageOutputPath>..\nuget</PackageOutputPath>
   </PropertyGroup>
 
   <ItemGroup Label="Additional nuget files">
-    <None Include="..\LICENSE.txt">
```

Comment:
I think this should be left so we continue to distribute a copy of the license in the NuGet package.

Reply:
added back in